ui/projects/etl-server/src/lib/pipelines/pipeline-list/pipeline-list.component.ts

```diff
@@ -62,5 +65,16 @@ export class PipelineListComponent implements OnInit {
     });
   }
 
+  selectChangeHandler (event: any) {
```

Comment:
This is also unrelated to this PR 😄 

Regardless, I think it's better to implement this as a `set` function on `selectedOption` instead of having a click handler.